twitter_classifier/scala/src/main/scala/com/databricks/apps/twitterClassifier/package.scala

```diff
@@ -0,0 +1,38 @@
+package com.databricks.apps
+
+package twitterClassifier {
+  import org.apache.spark.SparkContext
+  import org.apache.spark.sql.SparkSession
+
+  object SparkSetup {
+    val spark = SparkSession
+      .builder
+      .appName(getClass.getSimpleName.replace("$", ""))
+      .master("local[*]")
```

Comment:
Why master is hardcoded? Don't we rather expect user to specify his master URL in command line when launching the application?